lib/sikuli/common.sikuli/common.py

```diff
@@ -93,6 +93,9 @@ def is_infolog_enabled(self):
     def set_mouse_delay(self, sec=0.5):
         Settings.MoveMouseDelay = sec
 
```

Comment:
(This is not set_type_delay related comment.)
I found an interesting thing. Maybe we should also set Settings.DelayBeforeMouseDown to 0! Its default value is 300ms which matches exactly what we measured before for MouseDown event! @askeing @MikeLien @ShakoHo , how do you think?

Reference: http://sikulix-2014.readthedocs.io/en/latest/scripting.html